test/e2e/framework/config_lifecycle_operations.go

```diff
@@ -4,10 +4,13 @@
 package framework
 
 import (
+	"encoding/json"
+	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/aunum/log"
```

Comment:
Can we update this to use the logger from `tanzu-plugin-runtime` as #85 is already merged?